layouts/index.html

```diff
@@ -105,18 +117,30 @@ <h2 class="title--answer">Mostly...</h2>
       </li>
       <li class="list__item--ok">
         Screen recording tool:
+        <a href="https://github.com/xlmnxp/blue-recorder">Blue Recorder</a>,
+        <a href="https://gitlab.gnome.org/GNOME/mutter">GNOME ScreenCast</a>,
```

Comment:
this links to mutter?

Reply:
Yes, it's the default screen recorder used in GNOME, it resides in Mutter.

Will remove.